src/components/tooltip/tooltip.jsx

```diff
@@ -21,56 +16,72 @@ export default function Tooltip({
   text,
   truncate,
 }) {
-  const [visible, setVisible] = useState(false);
+  // size of the triangle in pixels; this should be the same as teh value in the linked stylesheet
+  // TODO is there a way to share the variable so its only in one place
```

Comment:
Shared across how many places?

CSS <=> CSS: use css variables
JS <=> JS: use exported variables
JS <=> CSS: use css variables, set variable in JS on DOM as `style={{ --le-var: '8px' }}`
JS <=> JS <=> CSS: export all the things!

Reply:
So it's basically what I mentioned here: https://github.com/mavenlink/design-system/pull/333#discussion_r661669634

JS and CSS are what need to share it - I tried using `style` but ran into issues. I might need your help in it.